test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Error.Tests.ps1

```diff
@@ -26,6 +26,17 @@ Describe 'Get-Error tests' -Tag CI {
 
         $out = Get-Error | Out-String
         $out | Should -BeLikeExactly '*InnerException*'
+
+        $err = Get-Error
+        $err.GetType().Name | Should -BeExactly 'ErrorRecord'
```

Comment:
it would be good to have the same tests for an exception, if possible

Reply:
Can't find a way to have a pure Exception in the test script.  If I type `1/d` interactively, I get a ParseException, but if I use `Invoke-Expression` in the Pester test, that cmdlet wraps it in an ErrorRecord.

Reply:
Maybe `Should -Throw -PassThru` helps?

Reply:
That doesn't help as it's already wrapped in an ErrorRecord by `Invoke-Expression`.  I tried using `pwsh -c`, but that ends up wrapping it as an ErrorRecord as well.  Only if I do `1/d` interactively in the console do I get a ParseException that isn't wrapped.